### ISSUE TITLE
Toggle Predicition Layer with [P] does not work during Object classification

### ISSUE THREAD

**Issue by tischi** [ilastik/ilastik#2421]
During the object and pixel classification workflow, when being at the object classification stage, pressing [P] does not do anything for me.

MacOS 10.14.6
ilastik 1.3.3post2

<img width="857" alt="image" src="https://user-images.githubusercontent.com/2157566/115146620-8d690100-a057-11eb-8198-456bdc28fd06.png">

**Comment by k-dominik** [comment]
Seems the reason is that this shortcut is not registered at all :/

**Comment by tischi** [comment]
Fair enough because Pixel prediction and Object prediction is different, but it would be very handy if it worked the same.

**Comment by k-dominik** [comment]
of course. In the meantime, I use the shortcut `i` a lot (to toggle between whatever you are looking at and raw-data-only display).
